generator/src/generator/namespace/switch.rs

```diff
@@ -808,7 +809,19 @@ fn emit_variable_size_switch_serialize(
             ext_params_arg_defs
         );
         out.indented(|out| {
-            serialize::emit_assert_for_switch_serialize(generator, switch, switch_expr_type, out);
+            //serialize::emit_assert_for_switch_serialize(generator, switch, switch_expr_type, out);
+
+            // eat the parameters for now
+            if !external_params.is_empty() {
```

Comment:
Yeah, this is definitely a bit rough, I just didn't want to go through with the breaking change that would be involved here or, alternatively, setting up the rest of the serialization logic to go along with this.